cypress/e2e/blueprints/explorer/workloads/deployments/deployment-create.ts

```diff
@@ -0,0 +1,240 @@
+export const createDeploymentBlueprint = {
+  apiVersion: 'apps/v1',
+  kind:       'Deployment',
+  metadata:   {
+    name:      'test-deployment-kubectl',
+    namespace: 'default',
+    labels:    { 'e2e-test': 'true' },
+  },
+  spec: {
+    replicas: 1,
+    template: {
+      spec: {
+        containers: [
+          {
+            name:      'nginx',
+            image:     'nginx',
+            resources: {
+              requests: {
+                cpu:    '200m',
+                memory: '512Mi'
+              },
+              limits: {
+                cpu:    '500m',
+                memory: '1Gi'
+              }
+            }
+          }
+        ]
+      },
+      metadata: {
+        labels:    { 'e2e-test': 'true' },
+        namespace: 'default'
+      }
+    },
+    selector: { matchLabels: { 'e2e-test': 'true' } }
+  }
+};
+
+export const deploymentCreateRequest = {
+  type:     'apps.deployment',
+  metadata: {
+    namespace: 'default',
+    labels:    { 'workload.user.cattle.io/workloadselector': 'apps.deployment-default-test-deployment' },
+    name:      'test-deployment'
+  },
+  spec: {
+    replicas: 1,
+    template: {
+      spec: {
+        restartPolicy: 'Always',
+        containers:    [
+          {
+            imagePullPolicy: 'Always',
+            name:            'container-0',
+            securityContext: {
+              runAsNonRoot:             false,
+              readOnlyRootFilesystem:   false,
+              privileged:               false,
+              allowPrivilegeEscalation: false
+            },
+            _init:        false,
+            volumeMounts: [],
+            __active:     true,
+            image:        'nginx'
+          }
+        ],
+        initContainers:   [],
+        imagePullSecrets: [],
+        volumes:          [],
+        affinity:         {}
+      },
+      metadata: {
+        labels:    { 'workload.user.cattle.io/workloadselector': 'apps.deployment-default-test-deployment' },
+        namespace: 'default'
+      }
+    },
+    selector: { matchLabels: { 'workload.user.cattle.io/workloadselector': 'apps.deployment-default-test-deployment' } }
+  }
+};
+
+export const deploymentCreateResponse = {
+  id:    'default/test-deployment',
+  type:  'apps.deployment',
+  links: {
+    remove: 'https://localhost:8005/v1/apps.deployments/default/test-deployment',
```

Comment:
Is the URL affecting in somehow the functionality? Because in that case we may want to replace with the `baseUrl`.